operators/osdk-getting-started.adoc

```diff
@@ -13,21 +13,12 @@
 toc::[]
 
 {nbsp} +
-This guide walks through an example of building a simple Memcached Operator
-(`memcached-operator`) using the `operator-sdk` CLI tool and
-`controller-runtime` library API. It also shows how to manage the new Operator's
-lifecycle from installation through updating to a new version.
-
-As an administrator of a Kubernetes-based cluster, you can accomplish this using
-two centerpieces of the Operator Framework:
-
-Operator SDK:: Assists developers in bootstrapping and building an Operator
-based on their expertise without requiring knowledge of Kubernetes API
-complexities.
-
-Operator Lifecycle Manager (OLM):: Helps Operator users install, update, and
-generally manage the lifecycle of all Operators and their associated services
-running across their clusters.
+As an administrator of a Kubernetes-based cluster (such as {product-title}),
```

Comment:
This reads funny to me. Sounds like it's saying that the guide is an administrator.

Reply:
Heh, I just re-read this a few minutes ago too. Awkward, will change.